det-lab/jupyterhub-deploy-kubernetes-jetstream
# Revisit NFS server deployment

- feat: increase NGINX body size
- feat: increase body size to 100MB
- redirect to det-lab
- Update README.md
- Update README.md
- feat: update the image to fedora atomic 29
- upgrade helm recipe to 0.9.0
- issuer for cert-manager installed with kubespray
- docs for HTTPS
- Revert "issuer for cert-manager installed with kubespray"
- Link new kubespray tutorial
- terminal pod script
- gitignore
- cvmfs gitignore
- magnum gitignore
- more gitignore
- set image Fedora-Atomic-29-JS-Latest
- NFS configuration files
- ssh server
- split volume creation
- split nfs service
- use deployments
- add toleration for magnum
- fix for latest NGINX helm chart
- fix read-only .ssh folder and contents permissions
- add link to github issue in config file
- avoid error for missing ssh folder
- set fs_gid to None
- Revert "set fs_gid to None"
- dask gateway files
- example grafana ingress
- setup https for grafana
- Update README.md
- the hacktoscaler
- No resources goes to stderr anyway
- toleration to run the hub on master
- install jupyterhub helm 0.11.1 by default
- node selector and tolerations under chp
- Update README.md
- update ingress to new version
- install stash script
- create secret script
- repository
- backup configuration
- ignore
- exclude jupyter local share from backup
- set the user to jovyan
- restore
- auto unlocking script
- backup templating
- customize the task name for each user
- remove unnecessary lines
- remove retention policy
- Revert "customize the task name for each user"
- retention policy is required
- restic tag usernames script
- wait 30s and check again, just to be sure we are not getting a transition
- don't tag if the username file is absent
- cluster issuer
- helm config for hadoop
- expose yarn script
- login yarn script
- link to tutorial
- more memory for production"
- Makefile to handle volumes
- default daskhub configuration
- import requirements and storage configuration
- install daskhub script
- set apitoken
- update for tutorial at https://zonca.dev/2022/01/dask-gateway-jupyterhub.html
- upgrade to JupyterHub Helm 1.2.0
- typo
- update stash
- just tag the last 10
- remove storage class and run on master
- change apiversion and name of cluster issuer
- switch to dask gateway 0.10.0
- install dask gateway from local checkout
- get credentials from env, add Zarr
- working deployment gateway dask zarr
- rename folder
- docker image of dask gateway with zarr package:
- support namespace
- convenience set environment script
- restic mount script
- also save restic repository
- monitor backup scripts
- do not lock
- switch to dask gateway 2022.4.0
- use post instead of get
- mariadb
- Patch cert-manager deployments; See #52
- make helm more verbose
- create user, fix permissions
- makefile
- link to source
- automatically create namespace
- manila secret
- ceph pod
- jupyterhub config
- Create README.md
- use JupyterLab by default
- build with no cache
- install jupyterlab
- GPU configuration file for JupyterHub
- rely on default storage class
